test/Sentry.Maui.Device.TestApp/Sentry.Maui.Device.TestApp.csproj

```diff
@@ -1,17 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <PropertyGroup Condition="'$(EnableMauiDeviceTestVisualRunner)' == 'true'">
```

Comment:
We could reverse this... only enable the maui device test visual runner property if the TFM is net9 or above.

That way we wouldn't have to fiddle with the TFMs in this csproj.

I think that would be cleaner and easier to reason about.

Reply:
Yeah, I was thinking we might have a compound condition for `EnableMauiDeviceTestVisualRunner`... something like:
```
Condition="'$(GITHUB_ACTIONS)' == 'true' and $([MSBuild]::IsTargetFrameworkCompatible('net9.0', $(TargetFramework)))"
```

So the visual tests would only be run locally and only when targeting net9.0 and later...

This csproj file wouldn't have to change then. It could target net8.0 and net9.0.

Reply:
I really don't like overcomplicating stuff on flags that only we're going to touch.  It will be forced to be net9.0 because of the TFMs in the target.  I can't use the original TFM setup in the device tests project.  I think we need to leave this one for now.

Reply:
Furthermore, if the visual runner flag gets left as true... the xharness device tests will fail on CI.... so we've got safety anyhow.

Reply:
It feels leaky to me. This is a bare bones test project... it's just a harness to run stuff. We shouldn't have logic in here related to to a specific assembly that it runs tests for.

Reply:
> I really don't like overcomplicating stuff

Yeah, that's what I'm trying to avoid as well. The above seems very complicated to me - and when combined with all the other flags we have I can see it being quite hard to grasp in the future. So I'm looking at ways to simplify it. 

I'll play around with it locally to see if what I'm proposing would even work and whether it would actually make things simpler or more complex (sometimes these things sound good in theory but the reality is something different).

Reply:
OK, I think I got it going... but man MS Build is painful to work with sometimes 😢 

This is hopefully a bit neater and it means you don't have to manually enable visual tests when running locally (they're always enabled locally when running the device tests against net9.0 and later targets).